packages/altair-fastify-plugin/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import { getDistDirectory, renderAltair, RenderOptions } from 'altair-static';
+import fp from 'fastify-plugin';
+import fastifyStatic from 'fastify-static';
+
+import type { FastifyPluginCallback } from 'fastify';
+
+export interface AltairFastifyPluginOptions extends RenderOptions {
+  /**
+   * URL to set as the server endpoint.
+   *
+   * By default is `/graphql`
+   */
+  endpointURL?: string;
+  /**
+   * URL to be used as a base for relative URLs and hosting needed static files.
+   *
+   * By default is `/altair/`
+   */
+  baseUrl?: string;
+  /**
+   * Path in which Altair will be accesible.
+   *
+   * By default is `/altair`
+   */
+  path?: string;
+}
+
+const fastifyAltairPlugin: FastifyPluginCallback<AltairFastifyPluginOptions> = (
+  fastify,
+  { path, ...renderOptions } = {},
+  done
+) => {
+  if (renderOptions.baseURL == null) {
```

Comment:
```suggestion
  if (renderOptions.baseURL === null) {
```

Reply:
You should be able to set these defaults in the argument. Something like:

`{ path = '/altair', endpointURL = '/graphql', baseURL = '/altair/',  ...renderOptions } = {}`

Reply:
The way it is right now allows me to just pass the whole options object to the renderOptions

Reply:
It just styling,  I just changed it to inline defaults.